sdks/python/apache_beam/yaml/readme_test.py

```diff
@@ -85,13 +86,46 @@ def guess_name_and_type(expr):
     output_schema = [
         guess_name_and_type(expr) for expr in m.group(1).split(',')
     ]
-    return next(iter(inputs.values())).pipeline | beam.Create(
-        [beam.Row(**{name: typ()
-                     for name, typ in output_schema})])
+    output_element = beam.Row(**{name: typ() for name, typ in output_schema})
+    return next(iter(inputs.values())) | beam.Map(
```

Comment:
Note: previously the output didn't derive from the input so no windowing was inherited.